astropy/units/tests/test_quantity_non_ufuncs.py

```diff
@@ -2357,7 +2353,7 @@ def test_merge_arrays(self, flatten):
     @pytest.mark.parametrize("flatten", [True, False])
     def test_merge_arrays_nonquantities(self, flatten):
         # Fails because cannot create quantity from structured array.
-        arr = rfn.merge_arrays((q_pv["p"], q_pv.value), flatten=flatten)
+        arr = rfn.merge_arrays((self.q_pv["p"], self.q_pv.value), flatten=flatten)
```

Comment:
You appeared to have fixed this test because now it is passing and failed the "strict xfail" check. Why was it marked as xfail to begin with?